### ISSUE TITLE
Target Microsoft.Extensions and Microsoft.AspNetCore in v2.1 (LTS)

### ISSUE THREAD

**Issue by prochnowc** [AppMetrics/AppMetrics#443]
Created as a follow-up issue from #390.

Since .NET Core / ASP .NET Core v2.1 is an LTS release most business apps will probably depend on v2.1 instead of v2.2. Currently AppMetrics targets v2.2 which forces all users to update to v2.2.

Furthermore AppMetrics seems to not use any v2.2 specific features so that a down-grade should be possible.
